docs/content/any/project/changelogs/2021-11-30.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Release 2021-11-30
+menuTitle: "2021-11-30"
+any: true
+description: >-
+  Changelog for Release 2021-11-30 (0.24.0) containing new features,
```

Comment:
Release should be `2021-12-01` (we're officially "releasing" it tomorrow) — filename should be updated too